UM/Settings/ContainerRegistry.py

```diff
@@ -141,6 +141,8 @@ def getEmptyInstanceContainer(self) -> InstanceContainer:
     #   that were already added when the first call to this method happened will not be re-added.
     def load(self) -> None:
         files = []
+        old_file_expression = re.compile("\\" + os.sep + "old\\" + os.sep + "\d+\\" + os.sep)
```

Comment:
I'd prefer to use `re.compile(r"\{sep}old\{sep}\d+\{sep}".format(sep = os.sep)` here since this makes it less of a string puzzle. Additionally it avoids needing to escape the slashes due to using "raw string" mode.